src/modules/projects/components/tables/ProjectExternalSubmissionsTable.tsx

```diff
@@ -122,15 +149,38 @@ const ProjectExternalSubmissionsTable = ({
 
   const valuesViewComponent = useMemo(() => {
     return (
-      <Card sx={{ p: 2 }}>
-        {loading && <Loading />}
-        {selected && (
-          <DynamicView
-            values={submissionValues}
-            definition={selected.definition.definition}
-          />
+      <>
+        <Card sx={{ p: 2 }}>
+          {loading && <Loading />}
+          {selected && (
+            <DynamicView
+              values={submissionValues}
+              definition={selected.definition.definition}
+            />
+          )}
+        </Card>
+        {selected?.clientId && selected?.enrollmentId && (
+          <Stack sx={{ mt: 2 }}>
+            <Alert severity='success'>
```

Comment:
should the alert be above the summary (dynamic view)? This is fine but I guess I'm more used to alerts being at the top of the page/modal

Reply:
Martha is reworking this dialog in a separate PR, so I was going to let her handle moving this around appropriately!

Reply:
I could go either way, I guess I have an inclination to put it in the same place as the "Mark reviewed" checkbox would go on an as-yet-unreviewed form. Maybe we can check with design about it?